src/test/ui/parser/let-binop.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    let a: i8 *= 1; //~ ERROR can't reassign to a uninitialized variable
```

Comment:
same as above (I figured that if I comment on the source I should point out the tests as well to save you the effort)